src/arithmetic/agg_ctx.h

```diff
@@ -17,6 +17,8 @@ struct AggCtx {
 	SIValue result;
 	int (*Step)(struct AggCtx *ctx, SIValue *argv, int argc);
 	int (*ReduceNext)(struct AggCtx *ctx);
```

Comment:
I think renaming this one from ReduceNext to finalize would make think a bit clearer, thoughts?

Reply:
done